diagnostic_feedback/quiz.py

```diff
@@ -0,0 +1,280 @@
+import logging
+import copy
+
+from xblock.core import XBlock
+from xblock.fields import Scope, String, List, Integer, Dict
+from xblock.fragment import Fragment
+from xblockutils.resources import ResourceLoader
+
+from .resource import ResourceMixin
+from .quiz_result import QuizResultMixin
+
+from .helpers import MainHelper
+from .validators import Validator
+from .sub_api import my_api
+from .data_tool import ExportDataBlock
+
+log = logging.getLogger(__name__)
+loader = ResourceLoader(__name__)
+
+
+@XBlock.wants('user')
+class QuizBlock(ResourceMixin, QuizResultMixin, ExportDataBlock):
+    """
+
+    """
+    BUZ_FEED_QUIZ_VALUE = "BFQ"
+    BUZ_FEED_QUIZ_LABEL = "Buzz Feed Quiz"
```

Comment:
Nit: there's no space in "BuzzFeed" and we don't want to imply endorsement. Maybe call it "BuzzFeed-style quiz" ?